insights/core/__init__.py

```diff
@@ -426,6 +427,120 @@ def _scan(self):
 
         cls.scan(result_key, _scan)
 
+    def _get_after(self, timestamp, time_format='%Y-%m-%d %H:%M:%S', lines=None):
```

Comment:
Hmmmm.  I suppose I wanted to require the subclass to provide the 'correct' method and I wanted to prevent the rule writer thinking about overriding the timestamp format.

But ... I can see that that's probably unnecessary.  And if it saves a bunch of copy-and-paste code, then so much the better.  In the cases where the subclass needs to change the timestamp format, then it can use super(ThisLogClass, self).get_after() ...

Let me try putting that together and see what I get.